Tests/kaas/plugin/README.md

```diff
@@ -23,17 +23,91 @@
 
    ```
 
-2. Install dependencies:
+2. **Install dependencies:**
 
    ```bash
    (venv) pip install pip-tools
    (venv) pip-compile requirements.in
    (venv) pip-sync requirements.txt
    ```
 
-3. Set environment variables and launch the process:
+3. **Create a Cluster**
+
+   To create a cluster, run:
 
    ```bash
-   (venv) export CLUSTER_PROVIDER="kind"
-   (venv) python run.py
+   (venv) python run_plugin.py create <plugin-name> ../clusterspec.yaml
    ```
+
+## Setting up and running the cluster-stacks plugin
+
+This section covers how to configure and run the `cluster-stacks` plugin to provision Kubernetes clusters and perform conformance tests.
+
+### Requirements for cluster-stacks plugin
+
+In addition to the core requirements, ensure the following are set up:
+
+* An **OpenStack** environment configured and accessible.
+* A `clouds.yaml` file defining OpenStack credentials and endpoints.
+* An installing the envsubst via this command:
+  ```bash
+  GOBIN=/tmp go install github.com/drone/envsubst/v2/cmd/envsubst@latest
+  ```
+
+### Environment Variables
+
+Before running the `cluster-stacks` plugin, you need to set up the following environment variable:
+
+- **GIT_ACCESS_TOKEN**: This token is required for Git operations, especially if your repository is private.
+
+To set the `GIT_ACCESS_TOKEN`, run the following command in your terminal:
+
+```bash
+export GIT_ACCESS_TOKEN=<your-github-token>
+```
+
+### Configuring clusterspec.yaml file
+
+The `clusterspec.yaml` file is used to set parameters for creating a Kubernetes cluster with the `cluster-stacks` plugin. This file allows you to specify details related to the cluster-stack, Git integration, and cluster configurations.
+
+### Mandatory Parameter
+
+The only mandatory parameter in `clusterspec.yaml` is `clouds_yaml_path` field, which points to the `clouds.yaml` file for OpenStack. If other parameters are omitted, the default values will be used.
```

Comment:
It’s mandatory because the plugin needs access to the OpenStack instance. However, we could set the default value of this variable to `~/.config/openstack/clouds.yaml`, since that's what you’re using most of the time.

Reply:
Yes, the file is mandatory, but the path shouldn't be, because we have the default value for that.